src/ostree.jsx

```diff
@@ -137,9 +140,11 @@ const Curtain = ({ state, failure, message, reconnect }) => {
     return (
         <EmptyState variant={EmptyStateVariant.full}>
             {icon}
-            <Title headingLevel="h5" size="lg">{title}</Title>
-            { message && <EmptyStateBody>{message}</EmptyStateBody> }
-            { (state === 'failed' && reconnect) && <Button variant="primary">{ _("Reconnect") }</Button> }
+            <EmptyStateHeader titleText={title} headingLevel="h5" />
+            <EmptyStateFooter>
```

Comment:
That's not correct here. EmptyStateBody should be out of the EmptyStateFooter.

The footer should contain the button only.